docs/src/content/docs/guides/i18n.mdx

```diff
@@ -145,6 +145,10 @@ In addition to hosting translated content files, Starlight allows you to transla
 
 English, Czech, French, German, Italian, Japanese, Portuguese, Dutch, Danish, Spanish, Turkish, Arabic, Norwegian, and Simplified Chinese translated UI strings are provided out of the box, and we welcome [contributions to add more default languages](https://github.com/withastro/starlight/blob/main/CONTRIBUTING.md).
 
+:::tip
+By default, regional variants (e.g. `pt-BR`) will receive the built-in translations for its base language (e.g. `pt`).
+:::
+
```

Comment:
I’m not sure this is the right place for this where a user is focused on _adding_ a translation.

```suggestion
```

It might make most sense in the [configuration reference section for `lang`](https://deploy-preview-475--astro-starlight.netlify.app/reference/configuration/#lang), something like this:

> The BCP-47 tag for this language, e.g. `"en"`, `"ar"`, or `"zh-CN"`. If not set, the language’s directory name will be used by default. Language tags with regional subtags (e.g. `"pt-BR"` or `"en-US"`) will use built-in UI translations for their base language if no region-specific translations are found.